### PR TITLE
Add CircuitSetup.us CC2652P2

### PR DESCRIPTION
**Pull request by digiblur**

### CODE REVIEW

File: docs/information/supported_adapters.md [hunk 1]
```diff
@@ -69,6 +69,13 @@ Zigbee2MQTT officially supports the following adapters:
     <td><a href="https://github.com/tube0013/tube_gateways">Flashing</a></td>
     <td><a href="https://www.tubeszb.com/">Tube's ZB Store</a></td>
   </tr>
+  <tr>
+    <td><img src="../images/circset_cc2652.jpg"></td>
+    <td>CircuitSetup's CC2652P2 USB Coordinator</td>
+    <td>CC2652P based USB connected adapter pre-programmed with Z-Stack</td>
+    <td><a href="https://github.com/Koenkk/Z-Stack-firmware/raw/master/coordinator/Z-Stack_3.x.0/bin/CC1352P2_CC2652P_launchpad_coordinator_20210120.zip">Coordinator</a><br/> <a href="https://github.com/Koenkk/Z-Stack-firmware/raw/master/router/Z-Stack_3.x.0/bin/CC1352P2_CC2652P_launchpad_router_20210128.zip">Router</a></td>
```
Comment (Koenkk) [inline docs/information/supported_adapters.md:76]: One column is missing, please add a link with the flashing instructions there.

Comment (digiblur) [reply docs/information/supported_adapters.md:76]: I didn't see any specific ones. Is there a standard set of cc2652p2 instructions to link?

**Comment by digiblur** [comment]
@Koenkk updated with flashing instructions link

**Comment by Koenkk** [comment]
Thanks!

**Pull request merged**